auth/src/main/java/com/firebase/ui/auth/data/model/Resource.java

```diff
@@ -43,12 +62,18 @@ public Resource(@NonNull T value) {
      *
      * @param exception error in computing the result
      */
-    public Resource(@NonNull Exception exception) {
+    Resource(@NonNull Exception exception) {
```

Comment:
Those static factory methods look great! Is there any reason to keep these constructors at all?